cernopendata_client/cli.py

```diff
@@ -243,6 +243,12 @@ def get_file_locations(server, recid, doi, title, protocol, expand, verbose):
         DOWNLOAD_RETRY_SLEEP
     ),
 )
+@click.option(
+    "--download-engine",
+    "download_engine",
+    type=click.Choice(["requests", "pycurl", "xrootdpyfs"]),
+    help="Engine to be used in downloading files [requests, pycurl, xrootdpyfs] [default=requests(for HTTP), xrootdpyfs(for XRootD)]",
```

Comment:
Download engine to use when downloading files. The available values are 'requests', 'pycurl', 'xrootdpyfs'. [default=requests (for HTTP protocol), xrootdpyfs (for XRootD protocol)]